apps/arweave/src/ar_data_sync.erl

```diff
@@ -721,7 +730,7 @@ handle_cast({join, RecentBI}, State) ->
 			PreviousWeaveSize = element(2, hd(CurrentBI)),
 			{ok, OrphanedDataRoots} = remove_orphaned_data(State, Offset, PreviousWeaveSize),
 			{ok, Config} = application:get_env(arweave, config),
-			[gen_server:cast(list_to_atom("ar_data_sync_" ++ ar_storage_module:label(Module)),
+			[gen_server:cast(name(Module),
```

Comment:
name/1 accepts StoreID so we need name(ar_storage_module:id(Module))

Reply:
thanks!

Reply:
https://github.com/ArweaveTeam/arweave/pull/638/commits/f796b4815b18b9670860fdd00d1797c75a6bc57b